.github/scripts/release.py

```diff
@@ -1,7 +1,7 @@
 
 import argparse, copy, hashlib, json, re, requests, os, shutil
 
-version = '0.1.0'
+version = '0.2.1'
```

Comment:
0.2.0?

Reply:
The old script with the error was 0.2.0, but since it has been reverted, it is now invisible. should I change to 0.2.0?

Reply:
If there was not really any release with 0.2.0 (as it was rolled back), then 0.2.0 should be the next or?

Reply:
fixed